roles/matrix-dynamic-dns/tasks/validate_config.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Fail if required settings not defined
+  fail:
+    msg: >-
+      You need to define a required configuration setting (`{{ item }}`).
+  when: "vars[item] == ''"
+  with_items:
+    - "matrix_dynamic_dns_username"
+    - "matrix_dynamic_dns_password"
+    - "matrix_domain"
+    - "matrix_dynamic_dns_provider"
+    - "matrix_dynamic_dns_mode"
+
+- name: Fail if dynamic dns mode is incorrect
+  fail:
+    msg: >-
+      matrix_dynamic_dns_mode needs to be set to 'daemon' or 'startup'
```

Comment:
This says `'daemon'` or `'startup'`, but the check below is for `'daemon'` and `'dhcp'`